classes/class-gf-securesubmit.php

```diff
@@ -2562,4 +2562,23 @@ private function get_hps_meta_data($feed, $entry, $form)
 
         return $metadata;
     }
+
+    protected function normalizeCountry($country)
+    {
+        switch (strtolower($country)) {
+            case 'us':
+            case 'usa':
+            case 'united states':
+            case 'united states of america':
+                return 'USA';
+            case 'ca':
+            case 'can':
+            case 'cana':
+            case 'cgg':
+            case 'canada':
+                return 'CAN';
+            default:
+                throw new Exception(sprintf('Country "%s" is currently not supported', $country));
```

Comment:
I don't think it's safe to include the input as is without something like filtervar() to cleanse it. Might be safest to leave the actual input out of the response string. sprintf offers no security for strings